transforms/transforms_test.go

```diff
@@ -362,6 +363,45 @@ func TestUniqueErrorForInvalidHeader(t *testing.T) {
 	}
 }
 
+func TestGroupBy(t *testing.T) {
+	transform := GroupBy(KeyIdentifier("group"))
+	input := []optimus.Row{
+		{"group": "1", "key": "a"},
+		{"group": "2", "key": "d"},
+		{"group": "3", "key": "g"},
+		{"group": "1", "key": 1},
+		{"group": "2", "key": 2},
+		{"group": "3", "key": 3},
+		{"group": "1", "key": 'b'},
+		{"group": "2", "key": 'e'},
+		{"group": "3", "key": 'h'},
+	}
+	expected := []optimus.Row{
+		{"1": []optimus.Row{input[0], input[3], input[6]}},
```

Comment:
I like it that way too https://github.com/Clever/optimus/commit/aba5a63feb08ffa81ede1a4bbe61d3e72e25f7b3